pkg/operator/ceph/csi/template/rbd/csi-rbdplugin.yaml

```diff
@@ -204,13 +217,22 @@ spec:
           volumeMounts:
             - name: plugin-dir
               mountPath: /csi
+            - mountPath: /var/lib/cephcsi/log/rbd-plugin
+              name: csi-log
           imagePullPolicy: {{ .ImagePullPolicy }}
         {{ end }}
       volumes:
         - name: plugin-dir
           hostPath:
             path: "{{ .KubeletDirPath }}/plugins/{{ .DriverNamePrefix }}rbd.csi.ceph.com"
             type: DirectoryOrCreate
+        - name: csi-log
+          hostPath:
+            path: /var/lib/cephcsi/log/rbd-plugin
```

Comment:
Is `/var/lib/cephcsi` a customizable dir?

Upstream users might not have write permissions to `/var/lib`, which is why `/var/lib/rook` is a customizeable dir?

@Madhu-1 does Ceph-CSI write to `/var/lib/cephcsi` in other places today?

Reply:
`/var/lib/cephcsi`  the paths are customizable using volume mounts, unless I am missing anything

Reply:
@BlaineEXE good point we don't write/have access to `/var/lib/cephcsi` today. @parth-gr  we need to have a flag like KUBELET_PATH.

Reply:
I didn't get your point @Madhu-1, `KUBELET_PATH` where it is defined

I see that specifying the path, kubelet creates the directory https://kubernetes.io/docs/concepts/storage/volumes/#hostpath-fileorcreate-example

Reply:
ROOK_CSI_KUBELET_DIR_PATH  https://github.com/rook/rook/blob/master/deploy/examples/operator.yaml#L159C5-L159C30 we need something like this

Reply:
SO the volume mount path will be,

```
hostPath:
            path: /{{ .KubeletDirPath }}/cephcsi/log/rbd-plugin
```
Like that?

Reply:
```yaml
hostPath:
            path: /{{ .CsiLogDir }}/cephcsi/log/rbd-plugin
```
something like this

Reply:
Done, thanks

Reply:
@BlaineEXE are you suggesting for `dataDirHostPath`, to make csi logs independent of ceph clusters

```
hostPath:
            path: /{{ .dataDirHostPath }}/cephcsi/log/rbd-plugin
```

Reply:
I believe the path should not include the preceding slash (`{{ .dataDirHostPath }}/cephcsi/log/rbd-plugin`), but yes. `dataDirHostPath` is the dir given that allows users to tell Rook what dir is writeable if the default isn't.

For the dedicated CSI operator, I think Rook can rely on users to change the value using CSI operator CR, but for now, `dataDirHostPath` is a workable stand-in.